lib/Conch/Validation/BiosFirmwareVersion.pm

```diff
@@ -3,12 +3,9 @@ package Conch::Validation::BiosFirmwareVersion;
 use Mojo::Base 'Conch::Validation';
 
 use constant name        => 'bios_firmware_version';
-use constant version     => 1;
+use constant version     => 2;
```

Comment:
In some places in this commit, the version number of the validation is incremented, even though only the description is being updated. In others, the version number is *not* incremented. Seems like we might want a unified approach in the PR. I'm not dogmatic about which path.

Reply:
The version number is not incremented when it is not necessary to do so -- when the description has only changed by trimming whitespace off the start and end, which has already been reflected in the database.  The version is only incremented when the db record will change upon the next run of `Conch::ValidationSystem::load_validations`.

Reply:
ah ok.